packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -61,18 +71,23 @@ export const useResizeHandle = (params: UseResizeHandleParams) => {
   const currentValue = React.useRef(UNMEASURED);
 
   const updateElementsAttrs = React.useCallback(() => {
-    wrapperRef.current?.style.setProperty(
-      variableName,
-      `${currentValue.current}px`
-    );
+    let a11yValue = getA11ValueText(currentValue.current);
+
+    // If relative mode is enabled, we actually have to measure the element,
+    // because the currentValue is just the px offset.
+    if (relative) {
+      const sizeInPx = elementDimension(elementRef.current, growDirection);
+      a11yValue = getA11ValueText(sizeInPx);
+    }
```

Comment:
```suggestion
    const a11yValue = relative
      ? // If relative mode is enabled, we actually have to measure the element,
        // because the currentValue is just the px offset.
        getA11ValueText(elementDimension(elementRef.current, growDirection))
      : getA11ValueText(currentValue.current);
```